tests/nnapi/specs/V1_2/log_softmax_nnfw.mod.py

```diff
@@ -0,0 +1,72 @@
+#
```

Comment:
I thought that `_nnfw` postfix means this test case is made by us, but it has Android Open Source Project copyright.

Reply:
@YiHyunJin 
If you just copied this without modification from android tests, please remove `_nnfw` from this file name.
If you create new test, please add our copyright text (`Copyright (C) 2020 Samsung Electronics Co., Ltd. All Rights Reserved`) above android copyright text.

Reply:
DONE